src/Simplex/Messaging/Server/Main.hs

```diff
@@ -303,7 +305,7 @@ smpServerCLI_ generateSite serveStaticFiles cfgPath logPath =
                           agreeSecret = True,
                           networkConfig =
                             defaultNetworkConfig
-                              { socksProxy = either error id <$> strDecodeIni "PROXY" "socks_proxy" ini,
+                              { socksProxy = either error id <$!> strDecodeIni "PROXY" "socks_proxy" ini,
```

Comment:
`either error id <$!> strDecodeIni` is everywhere and continues to spread.
Move to the error/forcing to the function itself (with `HasCallStack`)?